modules/phase_field/test/tests/feature_volume_vpp_test/tests

```diff
@@ -1,101 +1,125 @@
-  [../]
+  []
 
-  [./percolation_test_parallel]
+  [percolation_test_parallel]
     type = 'CSVDiff'
     input = 'percolation_test.i'
     csvdiff = 'percolation_test_out_features_0000.csv'
 
     issues = "#13169"
-    requirement = "The FeatureVolumeVectorPostprocessor shall output whether a percolated pathway exists between
-                    specified primary_percolation_boundaries and secondary_percolation_boundaries."
+    requirement = "The FeatureVolumeVectorPostprocessor shall output whether a percolated pathway "
+                  "exists between
+                    specified primary_percolation_boundaries and "
+                  "secondary_percolation_boundaries."
     design = "/FeatureVolumeVectorPostprocessor.md"
 
     min_parallel = 2
     prereq = 'percolation_test'
-  [../]
+  []
 
-  [./boundary_area_2D_test]
+  [boundary_area_2D_test]
     type = 'CSVDiff'
     input = 'boundary_area_2D.i'
     csvdiff = 'boundary_area_2D_out_features_0000.csv'
 
     issues = "#13202"
-    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied boundary
-                    by each feature by integrating the corresponding order parameter on the boundary."
+    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied "
+                  "boundary
+                    by each feature by integrating the corresponding "
+                  "order parameter on the boundary."
     design = "/FeatureVolumeVectorPostprocessor.md"
 
     min_parallel = 2
-  [../]
+  []
 
-  [./boundary_area_2D_single_test]
+  [boundary_area_2D_single_test]
     type = 'CSVDiff'
     input = 'boundary_area_2D_single.i'
     csvdiff = 'boundary_area_2D_single_out_features_0000.csv'
 
     issues = "#13202"
-    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied boundary
-                    by each feature by calulating the area/length of boundary elements."
+    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied "
+                  "boundary
+                    by each feature by calulating the area/length of "
+                  "boundary elements."
     design = "/FeatureVolumeVectorPostprocessor.md"
 
     min_parallel = 2
-  [../]
+  []
 
-  [./boundary_area_3D_test]
+  [boundary_area_3D_test]
     type = 'CSVDiff'
     input = 'boundary_area_3D.i'
     csvdiff = 'boundary_area_3D_out_features_0000.csv'
 
     issues = "#13202"
-    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied boundary
-                    by each feature by integrating the corresponding order parameter on the boundary."
+    requirement = "The FeatureVolumeVectorPostprocessor shall calculate coverage of a supplied "
+                  "boundary
+                    by each feature by integrating the corresponding "
+                  "order parameter on the boundary."
```

Comment:
not sure what s up but you have these extra line breaks on all these requirements

Reply:
`hit format` is what's up :-D

Reply:
is that a command?
it works just as poorly as clang check with strings

Reply:
Wait... weren't you the guy who opened the PR to `hit format` all inputs in MOOSE? #17594

Reply:
This instance is particularly bad because the string already had a manual break. I'll fix this one (and the ones like it). But other than that, we should encourage canonical formatting of HIT files using `hit format`.

Reply:
I didnt use hit format, just good old `sed`.
Ok thanks.